src/core_codemods/combine_startswith_endswith.py

```diff
@@ -17,36 +18,54 @@ class CombineStartswithEndswith(SimpleCodemod, NameResolutionMixin):
     def leave_BooleanOperation(
         self, original_node: cst.BooleanOperation, updated_node: cst.BooleanOperation
     ) -> cst.CSTNode:
-        if not self.filter_by_path_includes_or_excludes(
-            self.node_position(original_node)
-        ):
-            return updated_node
-
         if self.matches_startswith_endswith_or_pattern(original_node):
-            left_call = cst.ensure_type(updated_node.left, cst.Call)
-            right_call = cst.ensure_type(updated_node.right, cst.Call)
+            if not self.filter_by_path_includes_or_excludes(
+                self.node_position(original_node)
+            ):
+                return updated_node
 
             self.report_change(original_node)
 
-            new_arg = cst.Arg(
-                value=cst.Tuple(
-                    elements=[
-                        cst.Element(value=left_call.args[0].value),
-                        cst.Element(value=right_call.args[0].value),
-                    ]
-                )
-            )
+            elements = []
```

Comment:
could you move ln29-ln49 into its own method so it would be nice and clean
` new_arg = cst.Arg(value=cst.Tuple(elements=self._make_elements()))`